src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -47,9 +48,10 @@ private PowerFxConfig(CultureInfo cultureInfo, EnumStoreBuilder enumStoreBuilder
         /// <summary>
         /// Initializes a new instance of the <see cref="PowerFxConfig"/> class.        
         /// </summary>
-        /// <param name="cultureInfo">Culture to use.</param>        
-        public PowerFxConfig(CultureInfo cultureInfo = null)
-            : this(cultureInfo, new EnumStoreBuilder().WithRequiredEnums(BuiltinFunctionsCore.BuiltinFunctionsLibrary)) 
+        /// <param name="cultureInfo">Culture to use.</param>
+        /// <param name="maxCallDepth"></param>        
+        public PowerFxConfig(CultureInfo cultureInfo = null, int maxCallDepth = 100)
```

Comment:
>int maxCallDepth = 100 [](http://example.com/codeflow?start=61&length=22)

Consider leaving this out of the public constructor for now. We may want to group a bunch of host-protection config under a single object, and if this is part of a signature, cleaning it up becomes a breaking change.

Reply:
Done